10-suspense-for-data-fetching/README.md

```diff
@@ -0,0 +1,148 @@
+# React Suspense for Data Fetching
+
+## Prerequisites
+Please go to https://github.com/msd-code-academy/react-workshop and clone the project. Then inside the project run:
+
+```
+git pull
+cd 10-suspense-for-data-fetching
+npm ci
+npm start
+```
+
+Development server should open on the port 1234 _(unfortunately hot reload doesn't work with parcel and react experimental)_.
```

Comment:
It took me a while to realise that there is no webpack. Since people are used to react create app, would it make sense to mention here that we use Parcel instead but that they don't have to worry about it?

Reply:
Yeah, I will add it.